### PR TITLE
add basic layout of main page that appears when user what to login (authorize in app)  or register on site

### PR DESCRIPTION
**Pull request by mv-v-7**

### CODE REVIEW

File: client/src/components/mainPage/LayoutMainPage.vue [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                        :src="require('@/assets/mainPage/office_worker.svg')"
+                        alt="collage office employee at work"
+                        height="400px"
+                        width="400px"
+                        class="mt-3"
+                    />
+                </VRow>
+            </VCol>
+        </VRow>
+        <VRow style="height: 150px;">
```
Comment (lenchvolodymyr) [inline client/src/components/mainPage/LayoutMainPage.vue:59]: Add class and set height for that class

Comment (lenchvolodymyr) [reply client/src/components/mainPage/LayoutMainPage.vue:59]: it relates for all style attributes you used

File: client/src/components/mainPage/LayoutMainPage.vue [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                    height="50px"
+                    width="50px"
+                />
+            </VCol>
+        </VRow>
+        <VRow class="fill-height">
+            <VCol class="secondary col-1"></VCol>
+            <VCol class="secondary col-5">
+                <VRow align="start" justify="start">
+                    Here place component for login or authorise
```
Comment (lenchvolodymyr) [inline client/src/components/mainPage/LayoutMainPage.vue:43]: Instead of this phrase use slot

Comment (lenchvolodymyr) [reply client/src/components/mainPage/LayoutMainPage.vue:43]: https://vuejs.org/v2/guide/components-slots.html

File: client/src/components/mainPage/LayoutMainPage.vue [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                        :src="
+                            require('@/assets/mainPage/icon_circle_letter_s.svg')
+                        "
+                        alt="logo of schedulia"
+                        height="50px"
+                        width="50px"
+                    />
+                    <h2
+                        class="text--primary ml-3 mt-1"
+                        style="color: #281AC8 !important;"
```
Comment (lenchvolodymyr) [inline client/src/components/mainPage/LayoutMainPage.vue:17]: you added constants for this, use constants

File: client/src/components/mainPage/LayoutMainPage.vue [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                    >
+                        Shedulia
```
Comment (lenchvolodymyr) [inline client/src/components/mainPage/LayoutMainPage.vue:19]: move all phrases to localization store/i18n/en.js

File: client/src/App.vue [hunk 5]
```diff
@@ -45,6 +12,14 @@
                     <VCol class="text-center">
                         <RouterLink to="/status">Status</RouterLink>
                     </VCol>
+                    <VCol class="text-center">
```
Comment (lenchvolodymyr) [inline client/src/App.vue:15]: get rid of top menu

File: client/src/components/mainPage/LayoutMainPage.vue (hunk 3 above)
Comment (lenchvolodymyr) [reply client/src/components/mainPage/LayoutMainPage.vue:17]: instead of text-primary, use primary--text

**Comment by mv-v-7** [comment]
Because of new commit to develop branch and requirement to use i18n I created more good version in another branch and close this pull request.

**Pull request closed without merging**